src/SignalR.Orleans/Core/IConnectionGroupGrain.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Orleans;
+using System.Threading.Tasks;
+
+namespace SignalR.Orleans.Core
+{
+    public interface IConnectionGroupGrain : IGrainWithStringKey
+    {
+        Task AddMember(string connectionId);
+        Task RemoveMember(string connectionId);
```

Comment:
Same as Add...

Reply:
Renamed to `Remove` since now its `ConnectionGrain`